src/transformers/models/beit/feature_extraction_beit.py

```diff
@@ -98,11 +98,11 @@ def __call__(
         self,
         images: ImageInput,
         segmentation_maps: ImageInput = None,
-        return_tensors: Optional[Union[str, TensorType]] = None,
+        return_tensors: Optional[Union[str, TensorType]] = "np",
```

Comment:
Why is this change necessary? In general changing the default value of a function is a breaking change and those are to avoid at all cost.

Reply:
Fixing the feature extractors (as shown in #15055) will be a breaking change for sure. Basically, we want feature extractors to return a NumPy array of shape `(batch_size, num_channels, height_width)` by default (if the user hasn't specified return_tensors).

I am wondering whether this should be something for v5 of the library (as it's quite a big change but a required one notwithstanding), or whether we want to have this fix now already.

Reply:
I have not seen any large discussion around this proposed change, did I miss something?